stable/mysql/README.md

```diff
@@ -60,8 +60,10 @@ The following table lists the configurable parameters of the MySQL chart and the
 | `extraVolumes`                               | Additional volumes as a string to be passed to the `tpl` function                            |                                                      |
 | `extraVolumeMounts`                          | Additional volumeMounts as a string to be passed to the `tpl` function                       |                                                      |
 | `extraInitContainers`                        | Additional init containers as a string to be passed to the `tpl` function                    |                                                      |
+| `allowEmptyRootPassword`                     | Allow to leave `root` password blank                                                         | `false`                                              |
 | `mysqlRootPassword`                          | Password for the `root` user. Ignored if existing secret is provided                         | Random 10 characters                                 |
```

Comment:
Could you add "unless `mysqlAllowEmptyPassword` is set to true"?

Reply:
fixed